pytorch_toolkit/object_detection/model_templates/custom-detection/tools/eval.py

```diff
@@ -17,8 +17,8 @@
 
 from ote import MODEL_TEMPLATE_FILENAME
 from ote.api import test_args_parser
-from oteod.args_conversion import convert_ote_to_oteod_test_args
-from oteod.evaluation.common import evaluate
+from ote.args_conversion import convert_ote_to_oteod_test_args
+from ote.evaluation.common import evaluate
```

Comment:
I guess when something not mmbased is added, we should refactor it again. What if we create subpackages ote.mmbased (or similar) that will contain contents of oteod.

@evgeny-izutov Will it fully cover your needs for mmaction?

@evgeny-izutov @DmitriySidnev @druzhkov-paul any thoughts about my proposal?

Reply:
MMAction is less ornate rather than MMDetection except several comments I've made. Regarding the proposal to extract mm-related code to sub-package - I think it's more about the aesthetic of a code because the different (non mm-) model will redefine train, eval, export functionality on the own way and simply don't use them from ote.